cmd/encryption-v1.go

```diff
@@ -739,10 +740,10 @@ func (li *ListPartsInfo) IsEncrypted() bool {
 // DecryptedSize returns the size of the object after decryption in bytes.
 // It returns an error if the object is not encrypted or marked as encrypted
 // but has an invalid size.
-// DecryptedSize panics if the referred object is not encrypted.
+// DecryptedSize throws error msg and exits (fatal) if the referred object is not encrypted.
 func (o *ObjectInfo) DecryptedSize() (int64, error) {
 	if !o.IsEncrypted() {
-		panic("cannot compute decrypted size of an object which is not encrypted")
+		logger.FatalIf(errors.New("Cannot compute decrypted size of an unencrypted object"), "")
```

Comment:
This message needs to be more clear.

Reply:
Kannappan and I've decided to keep it as is.